src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -39,7 +39,6 @@ interface PlanListProps {
 /** Simple component that loads the new plan form and allows you to create a new plan */
 const PlanDefinitionList = (props: PlanListProps) => {
   const { fetchPlans, plans, service } = props;
-  const [loading, setLoading] = useState<boolean>(true);
```

Comment:
@p-netm do you need to create an issue for all other components that used this approach?

Reply:
#554